rubin_sim/skybrightness/interp_components.py

```diff
@@ -381,8 +381,11 @@ def __init__(self, mags=False, dark_sky_mags=None, fit_results=None):
             bp_temp.resample_bandpass(
                 wavelen_min=self.solar_spec.wavelen.min(),
                 wavelen_max=self.solar_spec.wavelen.max(),
-                wavelen_step=0.2,
+                wavelen_step=self.solar_spec.wavelen[1] - self.solar_spec.wavelen[0],
             )
+            # Force wavelengths to be identical so
+            # it doesn't try to resample again later
+            bp_temp.wavelen = self.solar_spec.wavelen
```

Comment:
If the solar spectrum had an irregular grid, this would be incorrect, so it's probably not a good example to set. (it's also worth noting that if I do "np.diff" on the solar spectrum wavelength array, I do get a few different values .. that look the same, but I'm assuming this machine precision stuff).